modules/sitemaps/sitemaps.php

```diff
@@ -706,6 +765,9 @@ function jetpack_sitemap_initialize() {
 	add_action( 'publish_page', 'jetpack_sitemap_handle_update', 12, 1 );
 	add_action( 'trash_post', 'jetpack_sitemap_handle_update', 12, 1 );
 	add_action( 'deleted_post', 'jetpack_sitemap_handle_update', 12, 1 );
+	// Clear sitemap cache if permalink structure is changed, used mostly because of multilingual plugins.
```

Comment:
Whenever we add an inline doc, we add a line break before so it's easier to spot when scanning the file. Can you please add it?